src/common/tusb_fifo.c

```diff
@@ -933,23 +933,26 @@ uint16_t tu_fifo_get_linear_read_info(tu_fifo_t *f, uint16_t offset, void **ptr,
   w = get_relative_pointer(f, w, 0);
   r = get_relative_pointer(f, r, offset);
 
+  // Copy pointer to buffer to start reading from
+  *ptr_lin = &f->buffer[r];
+  *ptr_wrap = f->buffer;
```

Comment:
You can add a zero check of `len_wrap` and `ptr_wrap` before assignment if you like, so they can leave to 0 if not used.

Reply:
I thought it is faster if we simply avoid this check. Anyway, if `len_wrap=0`, then ptr_wrap is not valid.